### ISSUE TITLE
File/Console title in iPhone portrait mode is left-aligned instead of centered

### ISSUE THREAD

**Issue by dgelessus** [omz/Pythonista-Issues#496]
Build 311017, iPhone SE, iOS 10.3.3.

In the latest beta build, when using my iPhone in portrait mode, the file name and "Console" title in the menu bars are left-aligned instead of centered. It's especially bad if you tap on the "Console" title - in the popup menu the word "Console" is literally touching the left border.

I assume this was somewhat intentional (at least in the editor) due to the additional toolbar buttons, which wouldn't fit otherwise if the title was centered. However in that case the title should be placed at a fixed position, not "as far to the left as possible". Currently if you tap on the title to open the popup menu, the title moves around as the toolbar buttons to the left of it change.

**Comment by omz** [comment]
This is intentional, but the console title doesn't touch the left border here (also iPhone SE, see screenshot below) – does this look different for you?

![img_a3bb1f90cec4-1](https://user-images.githubusercontent.com/82469/34292411-155f7e9a-e701-11e7-823a-4599940209b3.jpeg)

**Comment by dgelessus** [comment]
iOS 10 vs. 11 thing possibly? Here's what it looks like for me.

![img_0919](https://user-images.githubusercontent.com/6641959/34293035-665a9dbe-e703-11e7-8dc8-eba40bf2def6.PNG)

**Comment by omz** [comment]
@dgelessus Yes, just checked this in the iOS 10 simulator, and this is definitely not how it's supposed to look. Thanks!

**Comment by omz** [comment]
@dgelessus I just realized that this happens on iOS 11 as well, I just had a slightly older build on my iPhone SE where I apparently hadn't introduced the bug yet...
